common/navbar.js

```diff
@@ -32,6 +32,9 @@
     ])
     .directive('navbar', ['ConfigUtils', 'ERMrest', 'UriUtils', '$rootScope', '$window', function(ConfigUtils, ERMrest, UriUtils, $rootScope, $window) {
         var chaiseConfig = ConfigUtils.getConfigJSON();
+        var dcctx = ConfigUtils.getContextJSON();
+
+        $rootScope.hideNavbar = dcctx.hideNavbar;
```

Comment:
This is not important either, but you don't need to attach this to rootScope anymore, you can attach it to the scope in the link function.